addons/im_livechat/static/src/legacy/widgets/public_livechat_window/public_livechat_window.js

```diff
@@ -179,8 +179,12 @@ const PublicLivechatWindow = Widget.extend({
      * @private
      * @param {Object} messageData
      */
-    _postMessage(messageData) {
-        this.trigger_up('post_message_chat_window', { messageData });
+    async _postMessage(messageData) {
+        try {
+            await this.messaging.publicLivechatGlobal.livechatButtonView.sendMessage(messageData);
+        } catch (reason) {
```

Comment:
as you no longer user `reason`, you must add an `_` in front of it or you'll upset the linter 👺

Reply:
done